WordPress/Classes/ViewRelated/Blog/Add Site/AddSiteAlertFactory.swift

```diff
@@ -49,10 +50,50 @@ class AddSiteAlertFactory: NSObject {
 
     private func createWPComSiteAction(handler: @escaping () -> Void) -> UIAlertAction {
         return UIAlertAction(
-            title: NSLocalizedString("Create WordPress.com site", comment: "Create WordPress.com site button"),
+            title: Strings.createDotComSite,
```

Comment:
`AddSiteAlertFactory` is now used only by `BlogListViewController` and will be removed together with it, so I didn't update this implementation.